make.jl

```diff
@@ -101,20 +101,13 @@ cp(joinpath(tmp_path, "assets"), joinpath(tmp_path, "_site", "assets"), force=tr
 
 repo = "github.com:TuringLang/turing.ml.git"
 
-deploy_config = GitHubActions(
-    "TuringLang/turing.ml", #github_repository::String
-    "push", #github_event_name::String
-    is_dev ? "refs/heads/master" : "refs/tags/$(ARGS[1])" #github_ref::String
-)
```

Comment:
Removing this avoids publishing on PRs.

Reply:
I don't think this can be removed - it seems this would break the current Github action setup that publishes both the stable and dev version of the docs in every run. A manual `deploy_config` is needed to be able to do this.

Generally, it would also be good to add previews for PRs to be able to review them. However, it this involves too many changes maybe it should be done in a separate PR.